<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>349</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_input">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(208, 251, 225);
font: 700 15pt &quot;Quicksand&quot;;</string>
        </property>
        <property name="text">
         <string>Initial investment</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(208, 251, 225);
font: 700 15pt &quot;Quicksand&quot;;</string>
        </property>
        <property name="text">
         <string>Monthly contribution</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="init_invest">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-left: 5px; </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="monthly_contrib">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-left: 5px; </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(208, 251, 225);
font: 700 15pt &quot;Quicksand&quot;;</string>
        </property>
        <property name="text">
         <string>Term</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(208, 251, 225);
font: 700 15pt &quot;Quicksand&quot;;</string>
        </property>
        <property name="text">
         <string>Interest rate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="term">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-left: 5px; </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="interest_rate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-left: 5px; </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rb_months">
          <property name="styleSheet">
           <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;</string>
          </property>
          <property name="text">
           <string>Months</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_years">
          <property name="styleSheet">
           <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;</string>
          </property>
          <property name="text">
           <string>Years</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rb_simple">
          <property name="styleSheet">
           <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;</string>
          </property>
          <property name="text">
           <string>Simple interest</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_compound">
          <property name="styleSheet">
           <string notr="true">  color: rgb(208, 251, 225);
  font: 500 15pt &quot;Quicksand&quot;;</string>
          </property>
          <property name="text">
           <string>Monthly compound</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_calculate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton {
background-color: rgb(208, 251, 225);
color: rgb(57, 75, 64);
  font: 500 15pt &quot;Quicksand&quot;;
border-radius: 15px;
border-style: solid;
border-top-color: white;
border-left-color: white;
border-bottom-color: rgb(208, 251, 225);
border-right-color: rgb(208, 251, 225);
border-width: 1px;
}

QPushButton::pressed { 
background-color: rgb(179, 218, 195);
border-style: solid;
border-top-color: rgb(179, 218, 195);
border-left-color: rgb(179, 218, 195);
border-bottom-color: rgb(208, 251, 225);
border-right-color: rgb(208, 251, 225);
border-width: 1px;
}

</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_clear">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  font: 500 15pt &quot;Quicksand&quot;;
border-radius: 15px;
background-color: rgb(83, 174, 127);
color: rgb(57, 75, 64);
border-style: solid;
border-top-color: rgb(94, 205, 149);
border-left-color: rgb(94, 205, 149);
border-bottom-color: rgb(83, 174, 127);
border-right-color: rgb(83, 174, 127);
border-width: 1px;
}

QPushButton::pressed { 
background-color: rgb(62, 135, 99);
border-style: solid;
border-top-color: rgb(62, 135, 99);
border-left-color:rgb(62, 135, 99);
border-bottom-color: rgb(94, 205, 149);
border-right-color: rgb(94, 205, 149);
border-width: 1px;
}


</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>231</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 251, 225);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_output">
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 700 15pt &quot;Quicksand&quot;;
  padding-right: 10px; </string>
       </property>
       <property name="text">
        <string>Ending balance</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="res_end_balance">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-right: 10px; </string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 700 15pt &quot;Quicksand&quot;;
  padding-right: 10px; </string>
       </property>
       <property name="text">
        <string>Total contributions</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="res_total_contribs">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-right: 10px; </string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 700 15pt &quot;Quicksand&quot;;
  padding-right: 10px; </string>
       </property>
       <property name="text">
        <string>Total interest</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="res_total_interest">
       <property name="styleSheet">
        <string notr="true">color: rgb(57, 75, 64);
  font: 500 15pt &quot;Quicksand&quot;;
  padding-right: 10px; </string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>pb_calculate</zorder>
  <zorder>pb_clear</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
